experimental/dds/tree2/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -38,14 +38,14 @@ const nodesCountDeep = [
 	[100, BenchmarkType.Measurement],
 ];
 
-const deepBuilder = new SchemaBuilder("sharedTree.bench: deep", jsonSchema);
+const deepBuilder = new SchemaBuilder("sharedTree.bench: deep", {}, jsonSchema);
```

Comment:
Having to pass this in when you have libraries isn't great. Maybe a config object would be better?

Reply:
I'm not convinced `const deepBuilder = new SchemaBuilder("sharedTree.bench: deep", {libraries: [jsonSchema]});` is better, but maybe? I don't really like any of the options and I did consider that one. Something to get rethought in API review later I guess.